README.md

```diff
@@ -589,6 +589,7 @@ The `asdf` core provides a [security policy](https://github.com/asdf-vm/asdf/sec
 | tfenv                         | [carlduevel/asdf-tfenv](https://github.com/carlduevel/asdf-tfenv)                                               | [![Build Status](https://github.com/carlduevel/asdf-tfenv/actions/workflows/build.yml/badge.svg?branch=main)](https://github.com/carlduevel/asdf-tfenv/actions/workflows/build.yml)                                               |
 | TFLint                        | [skyzyx/asdf-tflint](https://github.com/skyzyx/asdf-tflint)                                                     | [![Build Status](https://travis-ci.org/skyzyx/asdf-tflint.svg?branch=master)](https://travis-ci.org/skyzyx/asdf-tflint)                                                                                                           |
 | tfmigrate                     | [dex4er/asdf-tfmigrate](https://github.com/dex4er/asdf-tfmigrate)                                               | [![CI](https://github.com/dex4er/asdf-tfmigrate/actions/workflows/ci.yml/badge.svg)](https://github.com/dex4er/asdf-tfmigrate/actions/workflows/ci.yml)                                                                           |
+| tfnotify                      | [jnavarrof/asdf-tfnotify](https://github.com/jnavarrof/asdf-tfnotify)                                           |                                                                                                                                                                                                                                   | [![Build Status](https://github.com/jnavarrof/asdf-tfnotify/actions/workflows/build.yml/badge.svg?branch=main)](https://github.com/jnvarrof/asdf-tfnotify/actions/workflows/build.yml) |
```

Comment:
You have added an extra column here with `||`, please update, reformat and push

Reply:
Thanks. It should be OK now.